menas/ui/service/EntityService.js

```diff
@@ -399,8 +438,26 @@ class MappingTableService extends DependentEntityService {
     return super.getByNameAndVersion(sName, iVersion, sModelPath, "#/mapping")
   }
 
+  cleanupEntity(oEntity) {
+    return {
+      name: oEntity.name,
+      description: oEntity.description,
+      schemaName: oEntity.schemaName,
+      schemaVersion: oEntity.schemaVersion,
+      hdfsPath: oEntity.hdfsPath,
+      title: oEntity.title,
```

Comment:
Remove `title`, that's used to show `Add` or `Edit` in the dialog's title.
This applies to all entities.